<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FnuevoEntrenamiento</class>
 <widget class="QDialog" name="FnuevoEntrenamiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo entrenamiento</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Al pulsar siguiente, se abrira el reproductor VLC &lt;br/&gt;para tomar muestras (frames) de entrenamiento, &lt;br/&gt;mediante la combinacion de teclas control+s.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="siguiente1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Siguiente</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccione el video para realizar el entrenamiento:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iNombreEntrenamiento">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selVideo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar video</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de entrenamiento:</string>
     </property>
    </widget>
    <widget class="QLabel" name="videoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="siguiente2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Siguiente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="iAgregarBg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar imagen de fondo</string>
     </property>
    </widget>
    <widget class="QListView" name="iBGview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>311</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="salir">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="entrenar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrenar</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="iOutputConsole">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ancho:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Largo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iLargo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>24</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iAncho">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>24</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iMuestras">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>muestras:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>stages:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iStages">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
